src/app/views/media/GridMediaCard.jsx

```diff
@@ -92,6 +90,8 @@ const GridMediaCard = ({
     }
   };
 
+  const MediaAcademyId = media.academy?.id;
```

Comment:
There is no need to create this variable, actually, you are using its value on line 105